src/Tests/Microsoft.Diagnostics.Monitoring.HostingStartup.UnitTests/ParameterCapturing/ObjectFormatting/Formatters/DebuggerDisplay/DebuggerDisplayFormatterTests.cs

```diff
@@ -28,7 +28,9 @@ private sealed class DebuggerDisplayClassWithNullField
         {
             public int Count { get; set; }
 
+#pragma warning disable CS8603 // Possible null reference return.
             public static string NullField => null;
```

Comment:
Why not just make this `string?` and not disable the warning?

Reply:
I thought it will impact the test, but on a second look you're right so I simplified the code.